docs/build.gradle

```diff
@@ -14,17 +14,13 @@
  * limitations under the License.
  */
 
-buildscript {
-  repositories {
-    jcenter()
-  }
-
-  dependencies {
-    classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.2'
-  }
+plugins {
+  id 'org.asciidoctor.convert' version '1.5.7'
+  id 'org.kordamp.gradle.livereload' version '0.2.1'
 }
 
 apply plugin: 'org.asciidoctor.convert'
+apply plugin: 'org.kordamp.gradle.livereload'
```

Comment:
Shouldn't we be using either the plugins dsl or the `apply plugin` stanzas not both?

Reply:
I don't know. I followed the official example https://github.com/asciidoctor/asciidoctor-gradle-examples/blob/master/asciidoc-to-html-livereload-example/build.gradle

Reply:
I'm fairly confident that the official example is wrong... could you test with one or the other and see?

Reply:
You are right. It seems that `plugins {}` will declare an apply the plugin at the same time. See https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block. Not complicated at all... (sight)